### PR TITLE
INT-451 Fail pipeline with policy action.

### PR DESCRIPTION
**Pull request by whyjustin**
https://issues.sonatype.org/browse/INT-451
https://youtu.be/BjSL4xNWaN4

### CODE REVIEW

File: src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorTest.groovy [hunk 1]
```diff
@@ -276,11 +276,30 @@ class IqPolicyEvaluatorTest
...
+          [new PolicyAlert(null, [new Action(Action.ID_FAIL)])], reportUrl)
+
+    when:
+      buildStep.perform(run, workspace, launcher, Mock(TaskListener))
+
+    then:
+      1 * iqClient.evaluateApplication('appId', 'stage', scanResult) >> policyEvaluation
+      1 * run.setResult(Result.FAILURE)
+
+    then:
```
Comment (fernau) [inline src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorTest.groovy]: you could use the and: label here

File: src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy [hunk 2]
```diff
@@ -234,6 +265,7 @@ class IqPolicyEvaluatorIntegrationTest
       }
   }
 
+
```
Comment (fernau) [inline src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy]: empty line

File: src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorTest.groovy (hunk 1 above)
Comment (whyjustin) [reply src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorTest.groovy]: e17e4e3

File: src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy (hunk 2 above)
Comment (whyjustin) [reply src/test/java/org/sonatype/nexus/ci/iq/IqPolicyEvaluatorIntegrationTest.groovy]: e17e4e3

**Pull request merged**